pkg/app/ops/insightcollector/collector.go

```diff
@@ -45,9 +45,8 @@ type Collector struct {
 	deploymentStore  deploymentStore
 	insightstore     insightstore.Store
 
-	applicationsHandlers []func(ctx context.Context, applications []*model.Application, target time.Time) error
-	// newlyCreatedDeploymentsHandlers []func(ctx context.Context, developments []*model.Deployment, target time.Time) error
-	newlyCompletedDeploymentsHandlers []func(ctx context.Context, developments []*model.Deployment) error
+	applicationHandlers []func(ctx context.Context, applications []*model.Application, target time.Time) error
```

Comment:
```suggestion
	applicationsHandler []func(ctx context.Context, applications []*model.Application, target time.Time) error
```
should it be like this? 🤔